docs/reference/configuration.md

```diff
@@ -28,15 +28,43 @@ This reference uses `.` to denote the nesting of values.
+  {: .note }
 * `logging.output` `(string : "-")` - A path or paths to write logs to. A `-` means the standard output, `=` means the standard error.
 * `logging.file_max_size_mb` `(int : 100)` - Output file maximum size in megabytes.
 * `logging.files_keep` `(int : 0)` - Number of log files to keep, default is all.
 * `actions.enabled` `(bool : true)` - Setting this to false will block hooks from being executed
-* `database.connection_string` `(string : "postgres://localhost:5432/postgres?sslmode=disable")` - PostgreSQL connection string to use
-* `database.max_open_connections` `(int : 25)` - Maximum number of open connections to the database
-* `database.max_idle_connections` `(int : 25)` - Sets the maximum number of connections in the idle connection pool
-* `database.connection_max_lifetime` `(duration : 5m)` - Sets the maximum amount of time a connection may be reused
+* ~~`database.connection_string` `(string : "postgres://localhost:5432/postgres?sslmode=disable")` - PostgreSQL connection string to use~~
+* ~~`database.max_open_connections` `(int : 25)` - Maximum number of open connections to the database~~
+* ~~`database.max_idle_connections` `(int : 25)` - Sets the maximum number of connections in the idle connection pool~~
+* ~~`database.connection_max_lifetime` `(duration : 5m)` - Sets the maximum amount of time a connection may be reused~~
+
+  **Note:** Deprecated - See `database` section 
+  {: .note }
+* `database` - Configuration section for the lakeFS key-value store database
+  + `database.type` `(string : ["postgres"|"dynamodb"])` - lakeFS database type 
+  + `database.postgres` - Configuration section when using `database.type="postgres"`
+    + `database.postgres.connection_string` `(string : "postgres://localhost:5432/postgres?sslmode=disable")` - PostgreSQL connection string to use
+    + `database.postgres.max_open_connections` `(int : 25)` - Maximum number of open connections to the database
+    + `database.postgres.max_idle_connections` `(int : 25)` - Sets the maximum number of connections in the idle connection pool
+    + `database.postgres.connection_max_lifetime` `(duration : 5m)` - Sets the maximum amount of time a connection may be reused
+  + `database.beta_dynamodb` - Configuration section when using `database.type="dynamodb"`
+    + `database.beta_dynamodb.table_name` `(string : kvstore)` -
+    + `database.beta_dynamodb.read_capacity_units` `(int : 1000)` - Throughput parameter for read capacity
+    + `database.beta_dynamodb.write_capacity_units` `(int : 1000)` - Throughput parameter for write capacity
+
+       **Note:** Refer to the following [AWS documentation](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Limits.html) for further information
+       {: .note }
+    + `database.beta_dynamodb.scan_limit` `(int : 0)` - Maximal number of items per page during scan operation
```

Comment:
0 means no limit or default by aws sdk?

Reply:
Not exactly - if not set (0) it will read up to 1MB of data. Added reference for documentation and removed default value